python-sdk/src/astro/extractors/extractor.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List, Optional
```

Comment:
The module path should be "lineage" instead of "extractors" I feel.

Instead of
```
python-sdk/src/astro/extractors/extractor.py
```

it should be:

```
python-sdk/src/astro/lineage/extractor.py
```

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1025/commits/7dad9a6fc4f0bcc5205f25fe51fb0b3f629a621e)